source/applications/advanced/color_balance.py

```diff
@@ -26,6 +26,19 @@ class MeanColor:
     blue: np.array
 
 
+def _frame2d_to_rgb(frame: zivid.Frame2D):
+    """Convert Zivid 2D frame to rgb numpy float array.
+
+    Args:
+        frame: 2D frame
+
+    Returns:
+        rgb: RGB image
```

Comment:
Perhaps note that this is a HxWx4 darray of rgba data (not just rgb).

Reply:
done